docs/develop-env-install.md

```diff
@@ -0,0 +1,19 @@
+# 1. SGX install
+## 1.1 driver
+## 1.2 SDK
+Select and download the SDK installing script from [here](https://download.01.org/intel-sgx/latest/linux-latest/distro), such as `sgx_linux_x64_sdk_2.17.100.3.bin`. 
+
+Run the script and specify the install directory `[path-to-install]`.  
+- The directory `[path-to-install]/sgxsdk/lib64` contains dynamic libraries, run the `ldconfig -v -n [path-to-install]/sgxsdk/lib64` to update the library reference.
+- The directory `[path-to-install]/sgxsdk/include` contains the header files, if compiler cannot find these files, use `export C_INCLUDE_PATH=$C_INCLUDE_PATH:[path-to-install]/include` to expose it.
+## 1.3 PSW
+### 1.3.1 Quote Generation and Verification
+Following the [guide](https://github.com/intel/SGXDataCenterAttestationPrimitives) to install libraries for quote generation and verification, such as `libsgx_dcap_quoteverify.so` 
+
+# 2. LibOS install
+
+## 2.1 Occlum
+Following the [guide](https://github.com/occlum/occlum) to install the occlum libOS.
```

Comment:
Does the default branch of occlum support enclave-cc? I remember `enclave-cc`  branch support this.

Reply:
Sure. Should use the `enclave-cc` branch.